packages/grafana-data/src/field/overrides/processors.ts

```diff
@@ -158,3 +158,15 @@ export interface StatsPickerConfigSettings {
    */
   defaultStat?: string;
 }
+
+export interface FieldNamePickerConfigSettings {
+  /**
+   * Optionally return true for fields that should be skipped
+   */
+  filter?: (field: Field) => boolean;
```

Comment:
normally the array filter function works the other way around, true for fields that should be preserved, so this would be the opposite of normal filter functions https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter

Reply:
ha -- I always get this backwards!   "filter" is a verb, and I assume true would mean get rid of it... but I know I am wrong so I then do the opposite, but then once you know it should be the opposite of what you think :infinity: :infinity: